### PR TITLE
finished the footer

### PR DESCRIPTION
**Pull request by gsysaath**
Take a look at the footer
The hover should be perfect
Screenshot 2020-11-16 at 17 28 19

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<footer class="page-footer font-small bg-primary pt-4">
+  <div class="container-fluid text-center text-md-left text-white">
+    <div class="row">
+      <div class="col-md-6 mt-md-0 mt-3">
+        <h5 class="text-uppercase">Vintage</h5>
+        <p>Vinyles is what noble music phisically look like.</p>
```
Comment (OlivierCalmels) [inline app/views/shared/_footer.html.erb:6]: Vinyles => Vinyls

Comment (awotawa) [inline app/views/shared/_footer.html.erb:6]: Vynils

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <a href="#!">A propos</a>
+          </li>
+          <li>
+            <a href="#!">En savoir plus sur l'entreprise</a>
+          </li>
+          <li>
+            <a href="#!">Fonctionnement</a>
+          </li>
+          <li>
+            <a href="#!">Conditions</a>
```
Comment (OlivierCalmels) [inline app/views/shared/_footer.html.erb:22]: You speack english ou tu parles francais?

**Comment by gsysaath** [comment]
I corrected all of the problems

**Review by awotawa — APPROVED** [review]
Sounds good to me 👍

**Pull request merged**